### ISSUE TITLE
Timeout error with whatWQPsites

### ISSUE THREAD

**Issue by lindsayplatt** [USGS-R/dataRetrieval#507]
I have installed the latest GitHub version of `dataRetrieval` to make sure that the errors I am seeing have not already been fixed. I am getting a timeout error on something that I thought should work since others have been able to run this line of code in the past. 

```
x <- whatWQPsites(siteType = "Lake, Reservoir, Impoundment", 
                  characteristicName = "Temperature, water")

Request failed [504]. Retrying in 1.7 seconds...
Request failed [504]. Retrying in 3.6 seconds...
For: https://www.waterqualitydata.us/data/Station/search?siteType=Lake%2C%20Reservoir%2C%20Impoundment&characteristicName=Temperature%2C%20water&zip=yes&mimeType=tsv

Error in getWebServiceData(obs_url, httr::write_disk(temp), httr::accept("application/zip")) : 
  Gateway Timeout (HTTP 504).
```

When I go to the web browser and search with these criteria, it says there are ~57K sites.

**Comment by ldecicco-USGS** [comment]
The mapper on the WQP does a header request I think to get those numbers, but if you actually try to get the data to return, there's a 504 timeout error as well:

![image](https://user-images.githubusercontent.com/1105215/75266071-73b38d00-57b7-11ea-92e3-28a23d5afef1.png)

I'd recommend looping through the states (either loop+bind_row or try a purrrr map)

**Comment by limnoliver** [comment]
An update on this issue - @ldecicco-USGS I was also having this same issue Lindsay described (used to work to call `whatWQPdata` for the nation so I could parse pulls) and tried your solution of looping through states. This also gives me a 504.

**Comment by ldecicco-USGS** [comment]
Do you know if it's some or all states? 504's tend to be WQP issues, not dataRetrieval (or a WAF issue....but from my perspective that's lumped into WQP's issue...)

```r
x <- whatWQPsites(statecode = "WI",
                  siteType = "Lake, Reservoir, Impoundment", 
                  characteristicName = "Temperature, water")
```
For instance, that just worked for me....maybe they updated some AWS thing and now the cloudfront is working again 🤞🤷‍♀️

If you've got a particular state/query, let me know, I'll pass it on if it seems non-dataRetrieval-y

**Comment by limnoliver** [comment]
Yeah, confident it's non-dataRetrievaly, but just wanted to follow up here in case someone came back to this issue. It works for many of the states, but not for some big ones (Florida).
